src/recon_buildblock/Parallelproj_projector/ForwardProjectorByBinParallelproj.cxx

```diff
@@ -126,7 +130,19 @@ actual_forward_project(RelatedViewgrams<float>& viewgrams,
     error("STIR wrapping of Parallelproj projectors current only handles projecting all data");
 
     viewgrams = _projected_data_sptr->get_related_viewgrams(
-        viewgrams.get_basic_view_segment_num(), _symmetries_sptr);
+        viewgrams.get_basic_view_segment_num(), _symmetries_sptr, false, viewgrams.get_basic_timing_pos_num());
+}
+
+static void
+TOF_transpose(float* STIR_mem, const std::vector<float>& mem_for_PP, const shared_ptr<const detail::ParallelprojHelper> _helper,
+              const long long offset, const long long num_lors_per_chunk)
+{
+  const auto num_tof_bins = static_cast<unsigned>(_helper->num_tof_bins);
+  for (unsigned tof_idx = 0; tof_idx < num_tof_bins; ++tof_idx)
+    for (long long lor_idx = 0; lor_idx < num_lors_per_chunk; ++lor_idx)
+      {
+        STIR_mem[offset + tof_idx * _helper->num_lors + lor_idx] = mem_for_PP[lor_idx * num_tof_bins + tof_idx];
+      }
```

Comment:
This a copy of the function in `BackProjectorByBinParallelproj.cxx`? Can it use a common function?

Reply:
not quite a copy